src/js/Router.js

```diff
@@ -15,7 +15,7 @@ export default class Router extends String {
         super();
 
         this._config = config ?? Ziggy ?? globalThis?.Ziggy;
-        this._config = { ...this._config, absolute };
+        this._config = { location: {}, ...this._config, absolute };
```

Comment:
This isn't strictly necessary, right? As long as we check for null/undefined when we access it below with something like `this._config.location?.host`?